src/components/TargetsMap/index.js

```diff
@@ -16,18 +20,39 @@ const TargetsMap = () => {
   const { latitude, longitude } = usePosition(true);
   const targets = useSelector(({ target: { values } }) => values);
   const topics = useSelector(({ topic: { values } }) => values);
+  const newTarget = useSelector(({ target: { value } }) => value);
+  const history = useHistory();
 
   const dispatch = useDispatch();
   useEffect(() => {
     dispatch(getTargets(1));
     dispatch(getTopics());
   }, []);
 
+  const handleNewTarget = location => {
+    dispatch(addTarget(location));
```

Comment:
Is just an alternative, but this info could skip redux and be passed directly.

Reply:
do you mean query string params?